### ISSUE TITLE
Links/Color contrast

### ISSUE THREAD

**Issue by aberkow** [codeforct/codeforct.github.io#3]
the color that's used for links (`#008AFF`) doesn't have sufficient color contrast to meet WCAG AA standards (minimum 4.5). it's currently 3.45. the link styles could be improved by

- adding an underline on body links for people with colorblindness. these underlines could be removed on hover/focus
- changing the color to something more along the lines of `#036fcc` which has a contrast of ~5.0
- the hover/focus color could be something more like `#2c769c` to improve contrast from ~3 to ~5
